dangerfile.js

```diff
@@ -1,24 +1,40 @@
 const { danger, warn } = require('danger')
+const toolbox = require('danger-plugin-toolbox')
 
-const hasValidDescription = github =>
-  github.pr.body && github.pr.body.length > 10
+// NOTE: jest snapshots make this useless because of the high amount of changes
+// TODO: find a way to filter out snapshot changes
+const checkFileChangeLimit = (threshold = 1000) => {
+  const { github: { pr } } = danger // prettier-ignore
 
-if (!hasValidDescription(danger.github))
-  warn(':grey_question: This PR does not include a description.')
+  if (pr.additions + pr.deletions <= threshold) return
 
-const hasReviewers = github => github.pr.requested_reviewers.length > 0
+  warn(':exclamation: This is a big PR. Can you break this into smaller PRs?')
+}
 
-if (!hasReviewers(danger.github))
-  warn(
-    ':exclamation: No Reviewer: Please add some teammates to review your PR.'
-  )
+const checkForReviewer = () => {
+  const { github } = danger
+  if (github.pr && github.pr.requested_reviewers.length > 0) return
 
-const isLargePR = github => {
-  const threshold = 1000
-  return github.pr.additions + github.pr.deletions > threshold
+  warn(':exclamation: Please add some teammates to review your PR.')
 }
 
-if (isLargePR(danger.github))
-  warn(
-    ':exclamation: Big PR: <i>Please split this into serveral smaller PRs to facilitate a better code review.</i>'
-  )
+void (async function main() {
```

Comment:
future support for async/await. didn't end up needing this in the end. remove or keep?